src/Nox.Generator/Application/Factories/EntityMapper.template.cs

```diff
@@ -30,7 +30,7 @@ public override void MapToEntity({{entity.Name}} entity, Entity entityDefinition
         dynamic? noxTypeValue;
     #pragma warning restore CS0168 // Variable is declared but never used
     {{ for key in entity.Keys -}}
-    {{- if key.Type == "Nuid" || key.Type == "AutoNumber" || key.Type == "DatabaseGuid" || key.Type == "Formula" -}}
+    {{- if key.Type == "Nuid" || key.Type == "AutoNumber" || key.Type == "Formula" -}}
```

Comment:
Could you please pay extran attention to this and similar lines? @ggrgovski @dejobratic @rochar 
Some of them were removed completely whereas some of them were updated as Guid

Reply:
This has already been changed if I'm not wrong in the latest version of the main branch.
However, Guid should work normally and call the Factory.

Reply:
Desired changes were applied